### PR TITLE
Update economy.js

### PR DESCRIPTION
**Pull request by aakashrajput**
added automatically badge fuction that the user who deservs it get the badge automatically; without getting it from any staff .
-  doing all the achievement checks here
  - because it would stress the server too much by giving badges manually

i am looking forward from u guys need more data for all badges so i can add them all then it will be lifesaver ;P

### CODE REVIEW

File: chat-plugins/economy.js [hunk 1]
```diff
@@ -621,7 +623,7 @@ exports.commands = {
 			}
 			return '';
 		};
-
+
```
Comment (panpawn) [inline chat-plugins/economy.js:626]: remove this trailing whitespace

**Comment by panpawn** [comment]
please use a proper commit name

**Comment by panpawn** [comment]
there isn't that much staff that this is needed

**Comment by aakashrajput** [comment]
BTW this badge system don't only work for staff badge online we can add many diff badges here.

**Pull request closed without merging**